campaigns.py

```diff
@@ -50,11 +50,32 @@ def template(app):
     env['template_contents'] = template.render(env.context)
 
 
+def clear_static_generated_templates():
+    """
+    Our various frontend applications use the wrapper.html.erb and
+    header_footer_only.html.erb templates. They get these templates
+    using the Slimmer gem. And this gem fetches these templates from
+    the static application, located using the ASSET_ROOT.
+
+    When static is deployed there are no generated wrapper.html.erb or
+    header_footer_only.html.erb templates. At the first request of
+    either of these, the application will generate the template. The
+    template will be placed in the public/template directory. From
+    that point on, the templates are served by nginx.
+
+    This function clears the public/template directory to force it to
+    be regenerated to include the emergency campaign.
+    """
+    for template in ('wrapper.html.erb', 'header_footer_only.html.erb'):
+        sudo('rm /var/apps/static/public/templates/{}'.format(template))
+
+
 def deploy_banner(application):
     execute(template, application)
     if application == 'frontend':
         remote_filename = '/var/apps/frontend/app/views/root/_campaign_notification.html.erb'
     elif application == 'static':
+        clear_static_generated_templates()
```

Comment:
Should we not run this command once we've uploaded the new content? Otherwise we might see a race condition.